src/rust/ru.readmanga/src/parser.rs

```diff
@@ -315,11 +315,16 @@ pub fn parse_chapters(html: &WNode, manga_id: &str) -> Result<Vec<Chapter>> {
 pub fn get_page_list(html: &WNode) -> Result<Vec<Page>> {
 	let parsing_error = helpers::create_parsing_error();
 
+	// aidoku::prelude::println!("get_pages_html: {}", html.text());
 	let script_text = html
 		.select(r"div.reader-controller > script[type=text/javascript]")
 		.pop()
 		.map(|script_node| script_node.data())
-		.ok_or(parsing_error)?;
+		.ok_or(parsing_error)
+		.and_then(|mut text| {
```

Comment:
use `map` here and just return `text` instead of `Ok(text)`.

also the previous version was actually fine. I thought that `replace_range` actually returned the string, but I was mistaken, so either way is really just the same.